### PR TITLE
Feature/nmr updates

### PR DESCRIPTION
**Pull request by stratkg**

### CODE REVIEW

File: R/normalize_nmr.R [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+                # edata_new <- log(edata_new)
+                edata_new <- apply(edata_new, 2, log)
+              }else{
+                stop("data_scale is not recognized as valid. Check attributes(omcisData)$data_info$data_scale")
+              }
+            }
+          }
+        }
+        
+        edata_new <- data.frame(edata_new) # convert back to data.frame since the apply statements above switch it to matrix, array
```
Comment (rarichardson92) [inline R/normalize_nmr.R:340]: I think we'll need to pass a check.names argument here?

File: R/edata_replace.R [hunk 2]
```diff
@@ -185,46 +274,140 @@ edata_replace.metabData <- function(omicsData, x, y){
...
+    sep = " "
+  ))
+  return(updated_data)
+}
+
+
 
-  message(paste(num_replaced, "instances of", x, "have been replaced with", y, sep=" "))
+# function for nmrdata
+#' @name edata_replace
```
Comment (rarichardson92) [inline R/edata_replace.R:346]: Add @export call and roxygenize for nmrData use

File: R/as.isobaricpepData.R [hunk 3]
```diff
@@ -1,29 +1,19 @@
 #' Convert Data to Appropriate pmartR Class
 #'
-#' Converts a list object or several data.frames of isobaric peptide-level data to an object of the class 'isobaricpepData'. Objects of the class 'isobaricpepData' are lists with two obligatory components \code{e_data} and \code{f_data}. An optional list component \code{e_meta} is used if analysis or visualization at other levels (e.g. protein) is also desired.
+#' Converts a list object or several data.frames of isobaric peptide-level data to an object of the class 'isobaricpepData'. Objects of the class 'isobaricpepData' are lists with two obligatory components \code{e_data} and \code{f_data}. An optional list component \code{e_meta} is used if analysis or visualization at other levels (e.g. protein) is also desired. !UPDATE THIS FUNCTION!
```
Comment (rarichardson92) [inline R/as.isobaricpepData.R:3]: Do we still need "update this function note?

File: R/plot_nmrnormRes.R [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    plot_title <- ifelse(is.null(title_plot), "Reference Metabolite Profile", title_plot)
+  }else{
+    if(is.null(metabolite_name)){
+      ylabel <- ifelse(is.null(y_lab), sample_property_cname, y_lab)
+      plot_title <- ifelse(is.null(title_plot), "Sample Property Profile", title_plot)
+    }
+  }
+  
+  p <- ggplot(data = data, aes(x = Sample, y = value))
+
```
Comment (rarichardson92) [inline R/plot_nmrnormRes.R:86]: could benefit from axis.text.x = element_text(angle = 90)

**Review by rarichardson92 — APPROVED** [review]

File: R/normalize_nmr.R (hunk 1 above)
Comment (stratkg) [reply R/normalize_nmr.R]: Done

File: R/edata_replace.R (hunk 2 above)
Comment (stratkg) [reply R/edata_replace.R:347]: Thanks, done!

File: R/plot_nmrnormRes.R (hunk 4 above)
Comment (stratkg) [reply R/plot_nmrnormRes.R:86]: Added this!

File: R/as.isobaricpepData.R (hunk 3 above)
Comment (stratkg) [reply R/as.isobaricpepData.R]: Good catch :)

**Pull request merged**